torch/_refs/__init__.py

```diff
@@ -534,7 +543,8 @@ def _ref(a: Tensor, *, out: Optional[Tensor] = None) -> Tensor:
 )
 
 erfinv = _make_elementwise_unary_reference(
-    prims.erf_inv, type_promotion=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT
+    prims.erf_inv, type_promotion=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
```

Comment:
Is there nowhere that we're mapping from references to torch ops? That seems like the better place to automatically map between ATen and reference ops.

Reply:
The mapping is implicit in the `erfinv = ` assignment. I suppose I could do some `locals()` tricks but this seemed... less invasive.

Reply:
I'm taking a note to fix this name mismatch, there's no reason to have it.